scripts/build/windows.bat

```diff
@@ -212,10 +212,7 @@ call %electron_packager% . %application_name%^
  --out=%output_build_directory%
 
 set package_output=%output_build_directory%\%package_name%
-
-if not "%arch%"=="%electron_arch%" (
-  move %output_build_directory%\Etcher-win32-%electron_arch% %package_output%
-)
```

Comment:
That seems to make sense, however @jviotti decided that it wasn't needed; was that just a case of not testing for 32-bit?